src/home_robot/home_robot/utils/image.py

```diff
@@ -10,6 +10,38 @@
 
 
 class Camera(object):
+    @staticmethod
+    def from_width_height_fov(
+        width: float,
+        height: float,
+        fov_degrees: float,
+        near_val: float = 0.1,
+        far_val: float = 4.0,
+    ):
+        """Create a camera given minimal information only. Fov is in degrees"""
+        horizontal_fov_rad = np.radians(fov_degrees)
+        focal_length_x = width / (2 * np.tan(horizontal_fov_rad / 2))
+        # focal_length_y = height / 2
+        focal_length_y = width / (2 * np.tan(horizontal_fov_rad / 2))
+        principal_point_x = (width - 1.0) / 2
+        principal_point_y = (height - 1.0) / 2
+        return Camera(
```

Comment:
Maybe not in this PR, but should this class be called PinholeCamera since it uses a pinhole model w/o any distortion?

Reply:
Yeah makes sense - although i was imagining you could override it, so might just note that pinhole is by default